scitt-scripts/create_signed_statement.py

```diff
@@ -96,18 +108,25 @@ def create_signed_statement(
                 },
             },
         },
+        HEADER_LABEL_PAYLOAD_HASH_ALGORITHM: -16,  # for sha256
+        HEADER_LABEL_LOCATION: payload_location,
     }
 
     unprotected_header = {
         X5chain: issuer.x5chain
     }
 
+    # now create a sha256 hash of the payload
+    #
+    # NOTE: any hashing algorithm can be used.
+    payload_hash = sha256(payload.encode("utf-8")).digest()
```

Comment:
Future enhancement idea: fetch the payload at the payload location and compare its hash with the specified payload file hash